.github/workflows/ci.yml

```diff
@@ -11,83 +11,158 @@ on:
 env:
   CARGO_TERM_COLOR: always
   DASEL_VERSION: https://github.com/TomWright/dasel/releases/download/v1.24.3/dasel_linux_amd64
+  RUSTFLAGS: "-D warnings"
+  RUST_VERSION: 1.61.0
+  FORC_VERSION: 0.13.0
 
 jobs:
-  build:
+  cancel-previous-runs:
     runs-on: ubuntu-latest
-
     steps:
       - name: Cancel Previous Runs
         uses: styfle/cancel-workflow-action@0.9.1
         with:
           access_token: ${{ github.token }}
 
-      - name: Checkout repository
-        uses: actions/checkout@v2
-
+  setup-test-projects:
+    needs: cancel-previous-runs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
       - name: Install toolchain
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: 1.61.0
+          toolchain: ${{ env.RUST_VERSION }}
           override: true
-
-      # Ensure CI is using the same minimum toolchain specified in fuels Cargo.toml
-      - name: Verify Rust Version
-        run: |
-          curl -sSLf "$DASEL_VERSION" -L -o dasel && chmod +x dasel
-          mv ./dasel /usr/local/bin/dasel
-          MIN_VERSION=$(cat packages/fuels/Cargo.toml | dasel -r toml 'package.rust-version')
-          RUSTC_VERSION=$(rustc --version -v | grep "release" | cut -d " " -f 2)
-          echo "Comparing minimum supported toolchain ($MIN_VERSION) with ci toolchain ($RUSTC_VERSION)"
-          test "$MIN_VERSION" == "$RUSTC_VERSION"
-
         # selecting a toolchain either by action or manual `rustup` calls should happen
         # before the cache plugin, as it uses the current rustc version as its cache key
       - uses: Swatinem/rust-cache@v1
 
-      - name: Install rustfmt
-        run: rustup component add rustfmt
-
       - name: Set git config
         run: |
           git config --global core.bigfilethreshold 100m
 
       - name: Install Forc
-        uses: actions-rs/cargo@v1
-        with:
-          command: install
-          args: forc --version 0.13.0
+        run: |
+          curl -sSLf https://github.com/FuelLabs/sway/releases/download/v${{ env.FORC_VERSION }}/forc-binaries-linux_amd64.tar.gz -L -o forc.tar.gz
+          tar -xvf forc.tar.gz
+          chmod +x forc-binaries/forc
+          mv forc-binaries/forc /usr/local/bin/forc
```

Comment:
Maybe move all binaries into `/usr/local/bin/`? That way CI can check for Forc formatting in a future PR easily.

Reply:
Nit, can be done in a future PR so as not to block.